lucille-core/src/test/java/com/kmwllc/lucille/indexer/SolrIndexerIntegrationTest.java

```diff
@@ -94,17 +96,19 @@ public void testDeleteById() throws Exception {
       indexer = new SolrIndexer(config, mockIndexerMessageManager, false, "solr");
       assertTrue(indexer.validateConnection());
 
+
       SolrQuery qr = new SolrQuery();
       qr.set("q", "*:*");
 
+      assertEquals("The test document should in the indexed.", 1, cluster.getSolrClient().query(COL, qr).getResults().size());
```

Comment:
thx for adding. pls review text: "The test document should in the indexed."

Reply:
Fixed.

Thanks, I guess I was cooked by the end of yesterday...